docs/comparison.md

```diff
@@ -43,7 +43,7 @@ All features are marked to indicate the following:
 | Devtools                                   | ✅                                  | ✅                                            | ✅                             |
 | Subscriptions                              | ✅                                  | ✅                                            | ✅                             |
 | Client-side Rehydration                    | ✅                                  | ✅                                            | ✅                             |
-| Polled Queries                             | ✅                                  | ✅                                            | ✅                             |
+| Polled Queries                             | 🛑                                  | ✅                                            | ✅                             |
```

Comment:
Cheers! 🙌 The only thing I'd propose is to use "🔶" because we removed the option since it requires very little code to implement in userland, i.e. "🔶" standing for "Supported and documented, but requires custom user-code to implement."

I believe what we need to do though is to make sure to readd it to the docs with our suggested code, which we haven't done yet because it'd be grouped with docs on the "Request Policy" and "Refocus" exchanges, which we in turn wanted to merge into a single one first

```suggestion
| Polled Queries                             | 🔶                                  | ✅                                            | ✅                             |
```